quesma/clickhouse/clickhouse.go

```diff
@@ -421,9 +421,18 @@ func (lm *LogManager) CreateTableFromInsertQuery(ctx context.Context, name strin
 	return nil
 }
 
+// This function takes an attributesMap and update it
+// with the fields that are not validated according to the inferred schema
+func addInvalidJsonFieldsToAttributes(attrsMap map[string][]interface{}, inValidJson types.JSON) {
```

Comment:
```suggestion
func addInvalidJsonFieldsToAttributes(attrsMap map[string][]interface{}, invalidJson types.JSON) {
```

here and in other places. The previous capitalization made it a bit confusing for me (it suggested "inValidJson = in valid JSON").